src/EventSubscriber/MenuLinkContentMigrateSubscriber.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\multiversion\EventSubscriber;
+
+use Drupal\Core\Database\Connection;
+use Drupal\Core\Menu\MenuLinkManagerInterface;
+use Drupal\multiversion\Event\MultiversionManagerEvent;
+use Drupal\multiversion\Event\MultiversionManagerEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * MenuContentLinkMigrateSubscriber class.
+ */
+class MenuLinkContentMigrateSubscriber implements EventSubscriberInterface {
+
+  /**
+   * @var \Drupal\Core\Database\Connection
+   */
+  protected $connection;
+
+  /**
+   * @var \Drupal\Core\Menu\MenuLinkManagerInterface
+   */
+  protected $menuLinkManager;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(Connection $connection, MenuLinkManagerInterface $menu_link_manager) {
+    $this->connection = $connection;
+    $this->menuLinkManager = $menu_link_manager;
+  }
+
+  /**
+   * A menu_tree database table should be rediscovered after enabling/disabling a menu_link_content entity.
```

Comment:
The comment is too long, should be refactored according to Drupal codding standards.